common-app-lib/src/cmr/common_app/services/search/query_validation.clj

```diff
@@ -24,9 +25,11 @@
                         result-format)]
     (when-not (get (supported-result-formats concept-type) result-format)
       (if-let [version (mt/version-of mime-type)]
-        [(format "The mime type [%s] with version [%s] is not supported for %ss."
-                 (mt/base-mime-type-of mime-type) version (name concept-type))]
-        [(format "The mime type [%s] is not supported for %ss." mime-type (name concept-type))]))))
+        ;; Some concepts aren't plural with just an 's'
```

Comment:
this comment is to remind us that other extensions may exist, or that is why we use a library?

Reply:
I don't think it is needed